### DISCUSSION TITLE
Most powerful extension that has ever come across the surface of this beautiful world that is Stable Diffusion.

### DISCUSSION THREAD

**Discussion by AloeVeraPurple** [Bing-su/adetailer#104] [Show and tell]
Where to begin ? This extension practically changed absolutely everything about my gens. I could just delete all my gens and start over from scratch thanks to this.

If used cleverly, with some patience to tweak every parameters to find the right tunes. You're able to gen the most incredible images without having to struggles with the downsides of SD's limits. Here are some of my best masterpieces that came out of using this beast.
![image](https://github.com/Bing-su/adetailer/assets/113100491/68f645b7-a492-4830-9ca1-aeb26afd0d0d)
![image](https://github.com/Bing-su/adetailer/assets/113100491/3e7e726a-c5d9-4a18-98e5-ad18ca6b48e9)
![image](https://github.com/Bing-su/adetailer/assets/113100491/52850316-e9f3-4347-b0bb-8edb26b41dbb)
![image](https://github.com/Bing-su/adetailer/assets/113100491/581944b5-bef9-4d62-aeb4-3a5ccf707003)
![image](https://github.com/Bing-su/adetailer/assets/113100491/4f005592-0229-4eb3-9665-fe98489635d5)
![image](https://github.com/Bing-su/adetailer/assets/113100491/06ee2611-baf5-4d31-a3c1-37d457addd43)

**I wanna take some time to also thank everyone that has contributed to this repo. Thank you all from the bottom of my heart..**

Bad hands, and bad faces.. are a topic of the past. I no longer need to waste much time with hiresfix. I can get clean results for prompt testing right away.

Thanks again and keep it up. I'm eagerly looking forward to see how this is gonna evolve...

Bests,
with love.

**Comment by andupotorac** [comment]
Would be great if you could provide your workflow for one of the images above, so one can see the before and after, and what prompts you used in ADetailer.

**Comment by CHollman82** [comment]
No offense but these aren't that great... you can do better without this extension, I have been for a long time. ADetailer is a great way to quickly inpaint faces without having to draw a mask... that's about it. It's a great extension, but if you are finding it drastically improving your images you were doing something wrong all along.

![342705559_792049975587520_6388208365714037502_n_2](https://github.com/Bing-su/adetailer/assets/3136566/c5a89144-1c95-444f-9c09-a5ab5d739f89)

If you're just talking about the hands/fingers there are checkpoints specifically trained to give good results, such as Deliberate v2. This image is old and nowhere near as good as my newest stuff but I was consistently getting hands like this back in 2022 with the Deliberate model:

![SD(1039404434)-ST(30)-CFG(13)-20230103043418](https://github.com/Bing-su/adetailer/assets/3136566/1e2d21e9-34e2-49fa-a0ca-6d8f07aecab5)

**Reply by CHollman82** [reply]
Having said that I have spent an absolute TON of time doing this, I ran a successful Patreon for a while selling images like this, and I've made over a million generations in less than a year with my RTX-4090, I have an external 12TB drive full of them... so... don't worry too much about it. I'm glad you found this extension so useful for you but in reality it is a matter of convenience and time saving... not expanding capability or quality.

**Reply by andupotorac** [reply]
Convenience is exactly what we're after. :)

**Comment by khanfahadsher** [comment]
@onegex1214 could you please be kind enough share a quick step by step process to generate such images? It would be a great help for a newbie like me :)

Cheers!
